docs/DevGuide/GitHubActions.md

```diff
@@ -136,27 +135,36 @@ useful to perform at least the following tests locally:
   longer require they be added so that IWYU gives no errors when run.
   As IWYU is still under development, we plan to investigate using it
   again in the future.
-
-## Travis setup {#travis-setup}
-
-* The `gcc Debug` build runs code coverage for each Travis build.
-
-## Troubleshooting {#travis-troubleshooting}
-
-* Occasionally, a build job will fail because of a problem with Travis
-(e.g. it times out).  Clicking on the circular arrow icon on the far
-right of the build job will restart it.  All build jobs can be
-restarted by clicking on the `Restart build` button to the right of
-the pull request description.
-* Travis caches some things between builds.  Occasionally this may
-cause a problem leading to strange build failures.  If you suspect
-this may be the case, get a SpECTRE owner to click on the `More
-options` button in the top right, choose `Caches` and delete the cache
-for your pull request.
+- The `gcc Debug` build runs code coverage for each GitHub Actions build.
+
+## Troubleshooting {#github-actions-troubleshooting}
+
+* Occasionally, a build job will fail because of a problem with GitHub Actions
+  (e.g. it times out).  On the `Checks` tab you can restart all or only the
+  failed jobs. In the top right corner there's a `Re-run jobs` menu, which also
+  has `Re-run failed jobs`. This button is `Cancel workflow` during the build
+  process.
```

Comment:
I think this is only for core-developers. I don't see the `Re-run jobs` menu or `Cancel workflow` for my PRs. Same for the rest of the things in the `Troubleshooting` section. I think only core-devs have access to manipulate workflows.